src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs

```diff
@@ -36,8 +37,8 @@ namespace Orleans.Providers.Streams.AzureQueue
     internal class AzureQueueBatchContainer : IBatchContainer
     {
         private EventSequenceToken sequenceToken;
-        private readonly List<object> events;
-        private readonly Dictionary<string, object> requestContext;
+        public readonly List<object> events;
```

Comment:
This is going to be completely unmanagable.
If we want internal runtime types to be Json serializable, we need to find a better/robust/testable way to enforce that. Otherwise, it will keep breaking. Look at the 5 times we already broke the pub sub state json serialization, since we used the approach of marking fields public. There are much better alternatives.

I suggest to open a separate issue and discuss it. I am sure people in our community have better expertise how to make sure their internal types are json-able.